hack/cluster-openshift.sh

```diff
@@ -42,7 +42,8 @@ get_downloader() {
 }
 
 # Change to the directory where this script is and set our env
-cd "$(dirname "${BASH_SOURCE[0]}")"
+SCRIPT_ROOT="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
Is there a reason you moved away from using BASH_SOURCE[0]? This script is explicitly a bash script (#!/bin/bash) so this is allowed.

Reply:
Yes, BASH_SOURCE[0] returns the relative path `.` instead of the absolute path.